include/lsst/afw/image/PhotoCalib.h

```diff
@@ -350,6 +350,20 @@ class PhotoCalib final : public table::io::PersistableFacade<PhotoCalib>, public
     MaskedImage<float> calibrateImage(MaskedImage<float> const &maskedImage,
                                       bool includeScaleUncertainty = true) const;
 
+    /**
+     * Return a un-calibrated image, with pixel values in ADU (or whatever the original input to
+     * this photoCalib was).
+     *
+     * Mask pixels are propagated directly from the input image.
+     *
+     * @param maskedImage The masked image with pixel units of nJy to uncalibrate.
+     * @param includeScaleUncertainty Include the uncertainty on the calibration in the resulting variance?
```

Comment:
I think what this really does (or ought to do) is more along the lines of "Remove uncertainty from this calibration that was previously propagated into the variance plane.  Should have the same value as the parameter of the same name when `calibrateImage` was called."